tests/unit/BaseTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * This class caters for the different PHPUnit versions between production 
+ * and development.
+ * To execute unit tests especially during development set an environment
+ * variable TEST_ENV=<environment> either as 'dev' or 'prod' :
+ * 
+ *   export TEST_ENV=<environment>
+ *   php codecept.phar run unit --debug
+ */
+$test_env = null !== getenv('TEST_ENV') ? getenv('TEST_ENV') : 'prod';
+
+if ($test_env !== 'dev') {
+	// php 5.6, production environment
```

Comment:
By production environment I guess you mean the environment in the CI. Can you please rename it here?